tests/fsharpqa/Source/Warnings/UpcastFunctionInsteadOfDowncast.fs

```diff
@@ -0,0 +1,9 @@
+// #Warnings
+//<Expects status="Error" span="(5,1)" id="FS3198">The conversion from Dictionary<obj,obj> to IDictionary<obj,obj> is a compile-time safe upcast, not a downcast. Consider using 'upcast' instead of 'downcast'.</Expects>
```

Comment:
this is a slightly different version of the hint. It#s not talking about operators